include/rmm/mr/device/pool_memory_resource.hpp

```diff
@@ -228,18 +229,17 @@ class pool_memory_resource final
   pool_memory_resource& operator=(pool_memory_resource&&)      = delete;
 
   /**
-   * @brief Get the upstream memory_resource object.
-   *
-   * @return const reference to the upstream memory resource.
+   * @briefreturn{rmm::device_async_resource_ref to the upstream resource}
    */
-  [[nodiscard]] const Upstream& upstream_resource() const noexcept { return *upstream_mr_; }
```

Comment:
This has just been added in the recent push so I am pretty sure it is safe to rename the method so that it is consistent with the other adaptors

Reply:
What PR added it?

Reply:
That was my initial memory_resource PR: https://github.com/rapidsai/rmm/pull/1095

Reply:
Ah, thought you meant someone else had added it.